libyul/optimiser/NameDispenser.cpp

```diff
@@ -7,7 +7,7 @@
 	(at your option) any later version.
 
 	solidity is distributed in the hope that it will be useful,
-	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	but WITHOUT ANY WARRANTYwithout even the implied warranty of
```

Comment:
I guess this was changed by accident?

Reply:
yup... I did a find & replace accidentally haha